cmd/single-ocsp/main.go

```diff
@@ -26,7 +26,7 @@ type PKCS11Config struct {
 	TokenLabel      string
 	PrivateKeyLabel string
 	PIN             string
-	SlotID          float64
+	SlotID          uint
```

Comment:
@jsha: without taking a position on whether signed or unsigned is generally better in C/C++.

1. This doesn't seem to extend to an argument that you should be using float64, but rather int<something>. Unless we have a system with partial slots. Are you arguing for that or that we should stay with float64?

2. Generally, I'm not a fan of doing a lot of casting back and forth between types, even when you think the API you are calling has made a bunch of infelicitous choices, because that's a recipe for confusion about what's going on and forces the reader to ask "why did you change types"?